rhizome/host/bin/setup-slice

```diff
@@ -0,0 +1,43 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../lib/slice_setup"
+
+unless (action = ARGV.shift)
+  puts "expected action as argument"
```

Comment:
This was following what's in setup-vm. If there is a better way, I am happy to update both places.

Reply:
Considering this accepts a different number of arguments for different subcommands, a more correct approach in terms of arguments would add significant complexity, so I don't think it's worth it.

As for the stdout vs stderr difference, it would definitely be important if this was a command a user would run, but considering this is for Rhizome, it's fine.

In short, I don't think you need to make changes for this.